lib/src/main/java/com/cloudbees/groovy/cps/impl/FunctionCallEnv.java

```diff
@@ -12,7 +13,7 @@
  */
 // TODO: should be package local once all the impls move into this class
 public class FunctionCallEnv extends CallEnv {
-    // TODO: delegate?
```

Comment:
Did you think about this comment by @kohsuke and decide it is no longer applicable and thus should be deleted?

Reply:
@jglick Yes, at least that was my assessment (I'd looked at using delegation in some of these cases and had to discard it).  

Would be curious what your read of this fairly-cryptic one-word comment is though, maybe you are interpreting it differently than I.